cli/tests/unit/serve_test.ts

```diff
@@ -2887,38 +2887,48 @@ Deno.test(
   },
 );
 
-// TODO(fbaltor): As it is this test should throw and fail
 // https://fetch.spec.whatwg.org/#dom-request-clone
 Deno.test({
   name: "Throw if locked",
-  ignore: true,
   permissions: { net: true },
   fn: async function shouldThrowIfBodyIsUnusableLocked() {
     const ac = new AbortController();
     const listeningPromise = deferred();
 
     const server = Deno.serve({
-      handler: (req) => {
+      handler: async (req) => {
         const _reader = req.body?.getReader();
 
-        req.clone();
+        try {
+          req.clone();
+        } catch (error) {
```

Comment:
Add a `fail();` immediately after `req.clone();` as well to ensure that this is throwing an error.

Reply:
Ok! The idea of this (and the previous) `fail()` is to force an error to make the test reliable? Since it should fail if the `clone()` is called but the `serve` probably is handling it, we need to force a fail, right?

Reply:
Correct. We want to make sure we fail in the right way. If this `clone` doesn't throw an exception, we need to detect that it didn't throw. This will help us ensure that it continues to throw as you've asserted here.